crypto/asn1/a_strex.c

```diff
@@ -343,8 +344,13 @@ static int do_print_ex(char_io *io_ch, void *arg, unsigned long lflags,
 
     if (lflags & ASN1_STRFLGS_SHOW_TYPE) {
         const char *tagname;
+        size_t tagname_len;
+
         tagname = ASN1_tag2str(type);
-        outlen += strlen(tagname);
+        tagname_len = strlen(tagname);
+        if (tagname_len > INT_MAX - 1)
```

Comment:
This is a bit silly really. `tagname_len` will never be greater than 17 as far as I can tell. Can't we just do the cast to `int` unconditionally?

Reply:
fixup pushed